apps/web/components/Organization/Wrapper/OrganizationWrapper.tsx

```diff
@@ -200,85 +200,213 @@ export const OrganizationHeader: React.FC<
   React.PropsWithChildren<HeaderProps>
 > = ({ organizationPage }) => {
   const { linkResolver } = useLinkResolver()
+  const namespace = useMemo(
+    () => JSON.parse(organizationPage?.organization?.namespace?.fields ?? '{}'),
+    [organizationPage?.organization?.namespace?.fields],
+  )
+  const n = useNamespace(namespace)
+  const { activeLocale } = useI18n()
+
+  const organizationLogoAltText = n(
+    'organizationLogoAltText',
+    activeLocale === 'is'
+      ? organizationPage.organization?.title + ' Forsíða'
```

Comment:
Here we are potentially creating a string that looks like this: 'undefined Forsíða'. You could do organizationPage.organization?.title ?? '' + ' Forsíða' to prevent that or somehow changing the conditional below.